en/user_guide/workflow.rst

```diff
@@ -0,0 +1,53 @@
+Workflow
+========
+
+Open orchestrated allows to have a publication workflow.
+It is a series of steps to achieve for publication of a node or a content before it can be displayed on Front Office.
+
+Status
+------
+
+Each node or content has a status per version, changing it allows to pass the publication workflow steps.
+The users can define the status which will make up their workflow as long as there is one initial and one final status.
+
+* An **initial** status is the first workflow status (assigned to a node or a content just after its creation).
+* A **final** status is the last workflow status (which allows the display on the Front Office).
+
+Status creation
+~~~~~~~~~~~~~~~
+
+* **Name** (required): status name
+* **Published**: if this status is final
+* **Initial**: if this status is initial
+* **Labels**: status label in all the Back Office languages
+* **Color** (required): status color
+
+.. image:: ../../images/status_form.png
+
+Roles
+-----
+
+Open Orchestra allows to create roles.
+
+* **Name**: role name
+* **Descriptions**: role description in all the languages of the Back Office.
+
+.. image:: ../../images/role_form.png
+
+See also `group management`_ to know how give roles to the users.
+
+Transitions
+~~~~~~~~~~~
+
+Transitions allow to update a node or a content from a status to another one.
```

Comment:
I feel like there's too many "allow" in our docs.
Maybe here you could say something like "A transition is a link between two status", and talk about the Workflow graph that results of this. A user guide describes more than graphic interface, it presents concepts behind to help the contributor.